controllers/scraper.go

```diff
@@ -0,0 +1,60 @@
+package controllers
+
+import (
+	"net/http"
+
+	form "go-crawler-challenge/forms/scrapper"
+	"go-crawler-challenge/services/scraper"
+
+	"github.com/beego/beego/v2/server/web"
+)
+
+// ScraperController operations for Dashboard
+type ScraperController struct {
+	BaseController
+}
+
+// NestPrepare prepares some configurations to the controller
+func (c *ScraperController) NestPrepare() {
+	c.actionPolicyMapping()
+}
+
+// URLMapping maps search controller actions to functions
+func (c *ScraperController) URLMapping() {
+	c.Mapping("Create", c.Create)
+}
+
+// actionPolicyMapping maps search controller actions to policies
+func (c *ScraperController) actionPolicyMapping() {
+	c.MappingPolicy("Create", Policy{requireAuthenticatedUser: true})
+}
+
+// Create handles keyword for scrapping
+// @Title Create
+// @Description create a new scrapping result
+// @Success 302 redirect to the dashboard page
+// @Failure 302 redirect to the dashboard page and show an error message
+// @router / [post]
+func (c *ScraperController) Create() {
```

Comment:
This action later will handle the CSV keywords upload, right? Because currently it handles only one keyword